src/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Leaderboard Project</title>
+    <link rel="stylesheet" href="/bootstrap.min.css">
```

Comment:
- [ ] Good job trying to inject some styling into your webpage. But currently, your styling seems not to take effect. 

This may be due to the fact that bootstrap is not installed as a dependency as I can't find it in your package.json or because you didn't add bootstrap CDN to your HTML file.

Kindly check out this [article](https://www.creative-tim.com/blog/web-design/add-bootstrap-html-guide/) to learn how to add bootstrap to an HTML file 

![image](https://user-images.githubusercontent.com/63473584/216441221-71c9a83f-e00c-4437-91fc-efdebe853e87.png)

Reply:
![image](https://user-images.githubusercontent.com/109720981/216581552-77b3e5de-7d4b-4ac8-9a24-8b9a816267f3.png)
Hello Sir, Is this what you are talking about? because the HTML file in src appears to have styling on my screen as shown in the screenshot attached.
Kindly correct me if I am wrong.
Regards.

Reply:
@TayyabKM, I didn't get that view from my end. I think you should add that screenshot to your pull request summary and then submit again. Another code reviewer will check it out.